Documentation/gettingstarted/minikube_intro.rst

```diff
@@ -11,11 +11,13 @@ Step 0: Install kubectl & minikube
 
 Then, boot a minikube cluster with the Container Network Interface (CNI)
 network plugin enabled plus the ``localkube`` bootstrapper since it contains
-``etcd`` >= ``3.1.0`` required by cilium.
+``etcd`` >= ``3.1.0`` required by Cilium. The ``CustomResourceValidation`` will
+allow Cilium to install the Cilium Network Policy validator into kubernetes
+(`more info <https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation>`_).
```

Comment:
This paragraph might be clearer with a "do this. These things are needed because" structure (since there are now 3 things we want them to include, spread over multiple sentences):
```
Boot a minukube cluster with the Container Network Interface (CNI) network plugin, the ``localkube`` bootstrapper, and CustomResourceValidation. The ``localkube`` bootstrapper provides ``etcd`` >= ``3.1.0``, a cilium dependency. ``CustomResourceValidation`` will allow Cilium to install the Cilium Network Policy validator into kubernetes (`more info <https://kubernetes.io/docs/tasks/access kubernetes-api/extend-api-custom-resource-definitions/#validation>`_)
```

Reply:
Done